### ISSUE TITLE
The code for the variants present in the paper

### ISSUE THREAD

**Issue by thedarkzeno** [facebookresearch/DiT#4]
Hello, it would be nice if you guys could release the code for the variants of the model like "DiT Block with Cross-Attention" and "DiT Block with In-Context Conditioning" for testing purposes, thanks.

**Comment by LucasSilvaFerreira** [comment]
Yes! I'm very curious how the cross-attention performs since the conditioning is a single token. Is this considering each column as a different "token"? @thedarkzeno , Do you know if I can use cross-attention on single tokens?
